cmd/commands/repo.go

```diff
@@ -258,6 +260,12 @@ func RunRepoBootstrap(ctx context.Context, opts *RepoBootstrapOptions) error {
 		return fmt.Errorf("failed to apply bootstrap manifests to cluster: %w", err)
 	}
 
+	// ALWAYS apply ArgoCD repo credentials secret (for both bootstrap and recovery)
+	log.G(ctx).Infof("applying argocd repository credentials...")
+	if err = opts.KubeFactory.Apply(ctx, manifests.argocdRepoCreds); err != nil {
```

Comment:
and then it can just be applied to the cluster together with the `namespace` and `applyManifests`, in line 259